pkg/build/admission/defaults/v1/types.go

```diff
@@ -0,0 +1,21 @@
+package v1
+
+import (
+	kapi "k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/unversioned"
+)
+
+// BuildDefaultsConfig controls the default information for Builds
+type BuildDefaultsConfig struct {
```

Comment:
is there a reason we don't define admission controller config types in the build api types.go instead of in their own types.go?  just convention?  Seems like it creates a lot of clutter (extra register logic, extra types files, etc)